backend/src/database/migrations/20230526011651-create-common-attachments-table.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('common_attachments', {
+      id: {
+        primaryKey: true,
+        allowNull: false,
+        type: Sequelize.DataTypes.UUID,
```

Comment:
I'm guessing we're using UUID here so that we have an ID we can easily pass to S3. 

However I wonder if using it for primary key might be bad for indexing and if it might not be better to use ULID instead. 

(Will insert links later -- have thought through sth similar for tx emails previously)

Reply:
For our usecase, none of the advantages of ULID is applicable (namely sorting using ID => we have created_at, no foreseeable need for partitioning), while the downside of it being not very widely supported yet (specifically not by sequelize) is applicable
=> Sticking with UUID v4